datasets/wikipedia/wikipedia.py

```diff
@@ -357,6 +357,516 @@
     "zu",
 ]
 
+MEDIA_ALIASES = {
```

Comment:
Could you please add a comment with the URL you get the aliases from?

Reply:
added -- it's slightly more complex because it's an API call for each language but I added enough that hopefully easy to update if ever needed.